uwblueprint/feeding-canadian-kids
# Add header buttons

## Notion Ticket Link

[Notion Ticket](https://www.notion.so/uwblueprintexecs/Add-logout-and-home-button-to-header-371d47eea6084b4f8ebc315d74f12882)



## Implementation Description
* Updated the header the match the Figma design specified in the ticket.
* Added the routing and functionality for each header button



## Steps To Test
1. Verify that the buttons match the Figma design
2. Verify that button functionality is correct



## What Should Reviewers Focus On?
* Just making sure the implementation is clean both design and experience wise


## Checklist
